### PR TITLE
Enable wireguard test for SLE

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://jira.suse.com/browse/SLE-14409
Verification: https://openqa.suse.de/tests/5406562# (only about the scheduling)

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -124,6 +124,7 @@ our @EXPORT = qw(
   load_extra_tests_y2uitest_gui
   load_extra_tests_kernel
   load_wicked_create_hdd
+  load_wireguard_tests
```
Comment (foursixnine) [inline lib/main_common.pm:127]: Do the wireguard tests run on their own test suite? if so, I'd vouch for moving this from main_common.pm to a yaml schedule instead...

**Review by SergioAtSUSE — COMMENTED** [review]
Why not taking the chance to port the test suite to YAML schedule?

**Comment by asdil12** [comment]
I was about do do the YAML thing but I'm not sure if I could simply move the mutex into the wireguard module.
For sure I can't create them within the YAML schedule.

File: lib/main_common.pm (hunk 1 above)
Comment (asdil12) [reply lib/main_common.pm:127]: I did that changes in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11900 - please have a look.

**Pull request closed without merging**